src/main/scala/firrtl/Parser.scala

```diff
@@ -44,20 +44,31 @@ case class InvalidEscapeCharException(message: String) extends ParserException(m
 
 object Parser extends LazyLogging
 {
+  private def time[R](name: String)(block: => R): R = {
```

Comment:
I agree it should be in Utils over the Parser.

I would prefer we eventually have a way this data can be collected but not necessarily printed out so it could be used in performance regressions, etc.